src/test/java/com/databerries/timezone/CountryCodeLookupTest.java

```diff
@@ -0,0 +1,28 @@
+package com.databerries.timezone;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+
+public class CountryCodeLookupTest {
+    private static CountryCodeLookup countryCodeLookup;
+
+    @BeforeClass
+    public static void initData() {
+        countryCodeLookup = CountryCodeLoader.createWithDefaultDataset();
+    }
+
+    @Test
+    public void testShouldBeInEurope() {
+        boolean isInEurope = countryCodeLookup.isTimezoneInEU("Europe/Paris");
+        Assert.assertThat(isInEurope, is(true));
+    }
+
+    @Test
+    public void testShouldNotBeInEurope() {
+        boolean isInEurope = countryCodeLookup.isTimezoneInEU("America/New_York");
+        Assert.assertThat(isInEurope, is(false));
+    }
```

Comment:
should we add a test for the `getCountryCode` method ?

Reply:
you right :+1: